boa/cli/boa.py

```diff
@@ -793,15 +789,15 @@ def main(config=None):
     parent_parser.add_argument("recipe_dir", type=str)
     parent_parser.add_argument("--features", type=str)
 
-    render_parser = subparsers.add_parser(
+    subparsers.add_parser(
         "render", parents=[parent_parser], help="render a recipe"
     )
-    convert_parser = subparsers.add_parser(
+    subparsers.add_parser(
         "convert",
         parents=[parent_parser],
         help="convert recipe.yaml to old-style meta.yaml",
     )
-    build_parser = subparsers.add_parser(
+    subparsers.add_parser(
         "build", parents=[parent_parser], help="build a recipe"
     )
 
```

Comment:
Line https://github.com/mamba-org/boa/pull/42/files#diff-861ef9dd4629a87f900b32efabb8d0c7880b87d61ded76ce22b4541344f0a401R952

It complains with `E722 do not use bare 'except'`, I don't know what kind of Exception is supposed to be raised so again I am not sure how to fix it